src/modules/data/blocks/ticket/sagas.js

```diff
@@ -25,7 +26,7 @@ import { wpREST } from '@moderntribe/common/utils/api';
 import { config, restNonce } from '@moderntribe/common/src/modules/utils/globals';
 import { TICKET_TYPES } from '@moderntribe/tickets/data/utils';
 import { blocks } from '@moderntribe/events/data';
-import { moment as momentUtil } from '@moderntribe/common/utils';
+import { toMoment, toDate, toTime24Hr } from '@moderntribe/common/utils/moment';
```

Comment:
This line is going to need to be reverted, it will mess up our externals since everything is put on the window